.semaphore/push-images/packaging.yaml

```diff
@@ -26,8 +26,10 @@ global_job_config:
       - export SECRET_KEY=$HOME/secrets/marvin.txt
       - export GCLOUD_ARGS='--zone us-east1-c --project tigera-wp-tcp-redirect'
       - export HOST=ubuntu@binaries-projectcalico-org
-      - sudo apt-get update -y
-      - sudo apt-get install -y moreutils patchelf
+      # Note that this override is for the docker containers we're building,
+      # and not the VM that we're running on right now
+      - export UBUNTU_REPO_OVERRIDE=ftp.uni-stuttgart.de
+      - install-package --no-install-recommends devscripts moreutils patchelf
```

Comment:
Is there a benefit from using `install-package`?  I've not heard of that, and there isn't a man page for it on my laptop.  Would rather stay with apt-get as I think more of us know that.

Reply:
Thanks for explaining, that makes sense.  Maybe a brief comment above that line to help future readers?